### ISSUE TITLE
Using ``--load-plugin`` instead of ``--load-plugins`` in CLI silently fail to load the plugin

### ISSUE THREAD

**Issue by Pierre-Sassoulas** [PyCQA/pylint#6810]
### Bug description

See [this comment](https://github.com/PyCQA/pylint/issues/6803#issuecomment-1145152401)

``pylint b.py --load-plugins=pylint.extensions.redefined_loop_name``
```
************* Module b
b.py:5:8: W2901: Redefining 'instrument' from loop (line 3) (redefined-loop-name)
```

### Command used

```shell
pylint b.py --load-plugin=pylint.extensions.redefined_loop_name
```

### Pylint output

```shell
No warning, ``redefined-loop-name`` or anything else.
```

### Expected behavior

Warning that load-plugin is not the right argument.

### Pylint version

```shell
2.14.0
```

**Comment by DanielNoord** [comment]
Not sure what the best approach is here. We allow abbreviations of arguments as `argparse` behaves differently on <3.8 (I think, but don't pin me on the version) if you don't allow abbreviations.
However, `load-plugins` is handled by an `if "load-plugins"` check. So while `argparse` recognises it as the abbreviation of an existing configuration option, it doesn't know how to handle it and our `if` statement doesn't allow catching such abbreviations.

**Comment by Pierre-Sassoulas** [comment]
Should we handle it a little like argparse maybe changing the condition to ``if "load-plugins".startswith(option) and len(option) > 6`` ?
